tests/api/test_canal_transit_entity.py

```diff
@@ -1,31 +1,40 @@
 from unittest import TestCase
 
-import json
-
 from vortexasdk.api.canal_transit import CanalTransitRecord
 
 
 class TestCanalTransitEntity(TestCase):
-    def test_serialize(self):
-        with open("tests/api/examples/canal_transit_entity.json", "r") as f:
-            serialized = json.load(f)
-            deserialized = CanalTransitRecord.parse_obj(serialized)
-
-            expected = CanalTransitRecord(
-                id="0004e233f16e15c0fcb3189332807e21306a3b3b06f0a778e3a3d9f276f233bd",
-                canal="canal1",
-                vessel_class=[
+    def test_serialize(self) -> None:
+        CanalTransitRecord.model_validate(
+            {
```

Comment:
nit: to keep the test closer to how it was should we pass a string as an argument and used `model_validate_json` instead?